objects/Cargo.toml

```diff
@@ -36,6 +36,7 @@ miden-test-utils = { workspace = true, optional = true }
 miden-verifier = { workspace = true }
 rand = { version = "0.8", optional = true }
 serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
+log = { version = "0.4", optional = true }
```

Comment:
Don't we need to specify the dependency in a feature above for this to work?  I'm guessing not but would be good to understand how this works.  Does the feature specified when building need to mirror the name of the dependency (`log`)?

Reply:
I suppose you mean a line like `log = ["dep:log"]` under the `dependencies` section. This is not necessary [ref](https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies)

> By default, this optional dependency implicitly defines a feature that looks like this: